docs/program/apis/cloud.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Managing Your Fleet with Viam's Cloud API"
+linkTitle: "Cloud Management"
+weight: 20
+type: "docs"
+description: "How to use the cloud API with Viam's client SDKs."
+tags: ["cloud", "sdk", "viam-server", "networking", "apis", "robot api", "cloud management"]
+---
+
+The cloud app API allows you to [manage your robot fleet](/manage/fleet/) with code instead of with the graphical interface of the [Viam app](https://app.viam.com/).
+Functionality includes:
+
+- create and manage organizations, locations, and individual robots
+- manage permissions and authorization
+- create and manage fragments
+
+{{% alert title="Support Notice" color="note" %}}
+
+Cloud app API methods are only available in the Python SDK.
+
+{{% /alert %}}
+
+## Establish a connection
+
+To use the Viam cloud app API, you first need to instantiate a [`ViamClient`](https://python.viam.dev/autoapi/viam/app/viam_client/index.html#viam.app.viam_client.ViamClient) and then instantiate an [`AppClient`](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient).
+See the following example for reference.
+You can find the location secret and the URL of your robot on the [Viam app](https://app.viam.com/):
```

Comment:
@andf-viam Okay what do you think of this section instead of the tip it was in before? And for this specific sentence, do you know if there's more specific info we can provide WRT where to find the URL and location secret? Do they have to go through the app for that?

Reply:
Looks great! I like your `object (named cloud)` clarification too!

For the address and locations I've been using something like [this](https://github.com/viamrobotics/docs/blob/9cb22785541bf49bcfee8b998b74b71b971f1122/docs/extend/modular-resources/examples/tflite-module.md?plain=1#L259-L262). This is a bit clunky for now, but will be changing with the upcoming org-level API keys coming down the pipeline.

Reply:
```suggestion
You can find the location secret and the URL of your robot on the **Code sample** tab in [Viam app](https://app.viam.com/):
```
Okay I think I want to keep this section somewhat streamlined since this page is probably for not-the-most-entry-level users anyway, and it's an aside to point them in the right direction versus a full-on tute. Is this enough?

Reply:
Sure, I'm OK with that for this usage! 👍

Reply:
EDIT: edited my above comment, think we need one more tweak!